cluster.go

```diff
@@ -447,6 +490,14 @@ process:
 	return resp
 }
 
+func (c *clusterClient) isSendToReplicas(cmd Completed) bool {
+	isSendToReplicas := false
+	if c.opt.SendToReplicas != nil {
+		isSendToReplicas = c.opt.SendToReplicas(cmd)
+	}
+	return isSendToReplicas
```

Comment:
```suggestion
	if c.opt.SendToReplicas != nil {
		return c.opt.SendToReplicas(cmd)
	}
	return false
```

Reply:
[600ecb0](https://github.com/redis/rueidis/pull/402/commits/600ecb060dcf12a25048d89d10783a8598158770) 
I change it